src/dotnet/project.json

```diff
@@ -26,6 +26,8 @@
     }
   },
   "dependencies": {
+    "NuGet.CommandLine.XPlat": "3.6.0-beta.1.msbuild.1",
```

Comment:
I don't think these references are needed anymore right?  We never actually load these assemblies into our process.  Instead we invoke MSBuild /t:Restore in a separate process.